scripts/dump_model_schemas.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from pydantic import BaseModel
+from scripts.models.patrol.patrol_schema import PatrolSchema
+from scripts.models.shortevent.short_event_schema import ShortEventSchema
+from scripts.models.thought.thought_schema import ThoughtSchema
+
+
+def dump_model_schema(config: BaseModel, output_name: str):
+    """Creates the JSON schemas based on the passed Pydantic model and writes it to a file"""
+    model_schema = config.model_json_schema()
+
+    with open(output_name, "w") as json_file:
```

Comment:
```suggestion
    with open(output_name, "w", encoding="utf-8") as json_file:
```
Pylint wants encoding to be explicit